### PR TITLE
PLU-269: add general ui fixes

### PR DESCRIPTION
**Pull request by m0nggh**
## Problem
Inconsistencies in our Plumber UI
TODO:
- [x] Check that dropdown for set up trigger looks ok after adding the `SingleSelect` dropdown PRs

## Details
- Standardise Test step + setup action button with ogp ds
- Adjust context menus icons etc.
- Standardise Alert Dialog texts
- Delete button color schemes change to critical
- Fix border radius for flow step
- Standardise UI for tiles page

## Tests
- [x] Still can delete and duplicate pipe
- [x] Still can delete tile
- [x] Still can delete column
- [x] Still can test step with old and new pipes

## Screenshots
Before

https://github.com/opengovsg/plumber/assets/65110268/fc2a48b5-f051-440b-8e90-f6c18191e281

After

https://github.com/opengovsg/plumber/assets/65110268/43ff982b-c3e8-4392-a9a5-cd4c0da2cee8

### CODE REVIEW

File: packages/frontend/src/components/FlowRow/FlowContextMenu.tsx [hunk 1]
```diff
@@ -125,22 +129,26 @@ export default function FlowContextMenu(props: FlowContextMenuProps) {
...
+          colorScheme="secondary"
           icon={<BiDotsHorizontalRounded />}
           variant="clear"
           onClick={(event) => {
             event.preventDefault()
             onMenuToggle()
           }}
         />
-        <MenuList>
+        <MenuList w="12.6875rem">
```
Comment (pregnantboy) [inline packages/frontend/src/components/FlowRow/FlowContextMenu.tsx]: erm...

File: packages/frontend/src/pages/Tile/components/TableFooter/DeleteRowsButton.tsx [hunk 2]
```diff
@@ -40,8 +40,8 @@ export default function DeleteRowsButton({
         variant="clear"
         size="xs"
         h="100%"
-        color="red.500"
-        leftIcon={<BsTrash />}
+        color="interaction.critical.default"
```
Comment (pregnantboy) [inline packages/frontend/src/pages/Tile/components/TableFooter/DeleteRowsButton.tsx]: colorScheme="critical"

**Review by pregnantboy — APPROVED** [review]
nits, otherwise lgtm

File: packages/frontend/src/components/FlowRow/FlowContextMenu.tsx (hunk 1 above)
Comment (m0nggh) [reply packages/frontend/src/components/FlowRow/FlowContextMenu.tsx]: ok i realised its cus i followed the figma design exactly but lemme change to `12.5rem` ba

**Pull request merged**